examples/demo-rollup/src/main.rs

```diff
@@ -1,5 +1,6 @@
 mod config;
 mod ledger_rpc;
+mod test_rpc;
```

Comment:
Lets feature-gate this so that it only compiles during testing
```suggestion
#[cfg(test)]
mod test_rpc;
```